0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -655,3 +658,18 @@ The URI of the connector.
     "connector": "http://usd-eur-connector.example"
 }
 ```
+
+## Class: ConnectOptions
+<code>class ConnectOptions</code>
+
+###### Fields
+| Type | Name | Description |
+|:--|:--|:--|
+| `Number` | timeout | milliseconds |
+
+### Fields
+
+#### id
+<code>**timeout**:Number</code>
+
+The number of milliseconds that the plugin should spend trying to connect before giving up.
```

Comment:
Probably should note that a falsy value (`null`, `undefined` and `0`) mean no timeout